src/components/client/donation-flow/steps/payment-method/TaxesCheckbox.tsx

```diff
@@ -50,15 +50,17 @@ export const TaxesCheckbox = () => {
           />
         </Grid2>
       </Grid2>
-      <Trans
-        t={t}
-        i18nKey="step.payment-method.alert.calculated-fees"
-        values={{
-          amount: moneyPublicDecimals2(amountWithoutFees.value),
-          fees: moneyPublicDecimals2(amountWithFees.value - amountWithoutFees.value),
-          totalAmount: moneyPublicDecimals2(amountWithFees.value),
-        }}
-      />
+      {!!amountWithFees.value && (
```

Comment:
Is this to assertain that the value is a non-zero number?  As in `!isNaN(amountWithFees.value) && amountWithFees.value > 0` 

It's hard to read. And also negative numbers give true value as well => `!!-1 === true`